src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.VisualBasic.targets

```diff
@@ -17,15 +17,18 @@ Copyright (c) .NET Foundation. All rights reserved.
   <PropertyGroup>
     <DefineConstants>$(DefineConstants),$(ImplicitFrameworkDefine)</DefineConstants>
   </PropertyGroup>
-  <ItemGroup>
+  <ItemGroup Condition=" '$(DisableImplicitFrameworkReferences)' != 'true' and '$(TargetFrameworkIdentifier)' == '.NETFramework'">
```

Comment:
The condition on TFM doesn't sound right. So netstandard libraries don't get these default imports?